### PR TITLE
LabWork5_Hristiyana_Shopova

### PR DESCRIPTION
**Pull request by hristiyanaSh**

### CODE REVIEW

File: Script.sql [hunk 1]
```diff
@@ -0,0 +1,32 @@
+select ifnull(IndepYear, 'N/A') as IndepYear from country ;
+
+select coalesce (indepyear, 'N/A') as IndepYear from country;
+
+select gnPold, GNP, coalesce (GNPold, GNP, 0) as GNPnew from country;
+
+
+
+#TASK1
+
```
Comment (sslavova) [inline Script.sql:10]: Here you need to take only the countries with N/A life expectancy. I would also add the names of these countries

File: Script.sql [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+select ifnull(LifeExpectancy, 'N/A') as LifeExpectancy from country ; 
+
+
+#TASK2
+
```
Comment (sslavova) [inline Script.sql:15]: The version with the case statements is correct, however the version with nested if statements has a syntax error. Fix the brackets

**Review by sslavova — CHANGES_REQUESTED** [review]

File: Script.sql (hunk 2 above)
Comment (sslavova) [reply Script.sql:15]: Brackets are still not fixed, please test your code before committing.

**Review by sslavova — CHANGES_REQUESTED** [review]

File: Script.sql (hunk 2 above)
Comment (hristiyanaSh) [reply Script.sql:16]: I corrected the brackets.

File: Script.sql [hunk 3]
```diff
@@ -24,7 +24,7 @@ as CountrySize from country ;
 select name, surfaceArea,
 case  
 when surfacearea > 1000000 then 'Large country'
-when SurfaceArea between 50000 and 1000000 then 'Medium country'
+when SurfaceArea between 50001 and 1000000 then 'Medium country'
 when surfaceArea between 20000 and 50000 then 'Small country'
 else 'Extra small country'
 end as CountrySize from country ;
```
Comment (sslavova) [inline Script.sql:30]: 1. Above query is ok now, but please apply the same logic in the other query with nested if statements.
2. Below code (line 33-55) is related to Lab work 4 - remove it from Lab work 5.

**Review by sslavova — CHANGES_REQUESTED** [review]

**Review by sslavova — APPROVED** [review]

**Review by sslavova — APPROVED** [review]

**Pull request closed without merging**